# request
uint16 gear

---
# response
uint16 gear_before
uint16 gear_after

